build.gradle

```diff
@@ -50,7 +50,7 @@ intellij {
 
     if ("DOKI" == System.getenv("DEV_ENV")) {
         setPlugins(
-            'io.acari.DDLCTheme:10.0.0'
+//            'io.acari.DDLCTheme:11.0.0'
```

Comment:
Does it seem that you forgot to uncomment this? Or you're planning to remove this?

Reply:
Nah, I just didn't want do as much work when I have a released build that supports 2020.3. However it will probably be 11.x. So I will just take being slightly annoyed having to put it back when it is ready.